src/pages/community/how-to-contribute.md

```diff
@@ -1,12 +1,122 @@
-# How to Contribute
+# How to Contribute to Apache Ozone
 
-**TODO [HDDS-9869](https://issues.apache.org/jira/browse/HDDS-9869) Fill in this page.**
+Thank you for your interest in contributing to Apache Ozone! This guide will help you understand the various ways you can contribute to the project and how to get started.
 
-This may end up similar to CONTRIBUTING.md in the Ozone repo, but should document all the ways to contribute outside of code as well.
+## Quick Links
 
-- Testing and filing issues
-  - This may link to the [Report an Issue](report-an-issue) page.
-- Contributing code
-  - Both Ozone backend and Recon UI/UX front end.
-- Contributing to the website
-- Graphic design work for logos or diagrams.
+- [GitHub Repository](https://github.com/apache/ozone)
+- [CONTRIBUTING.md](https://github.com/apache/ozone/blob/master/CONTRIBUTING.md) - Detailed contribution guidelines
+- [Newbie Jira Issues](https://issues.apache.org/jira/issues/?filter=12353868) - Good first issues for new contributors
+- [GitHub Discussions](https://github.com/apache/ozone/discussions) - Engage with the community
+- [Ozone Website](https://ozone.apache.org/) - Project documentation
+- [Slack Channel](http://s.apache.org/slack-invite) - Join #ozone on ASF Slack
+
+## Ways to Contribute
+
+Apache Ozone welcomes contributions in many forms, not just code. Here are various ways you can contribute:
+
+### 1. Code Contributions
+
+#### Finding Issues to Work On
+
+- **Newbie Issues**: Start with [beginner-friendly issues](https://issues.apache.org/jira/issues/?filter=12353868) that are good entry points
+- **All Open Issues**: Browse [unassigned Jira issues](https://s.apache.org/OzoneUnassignedJiras) to find something that matches your interests and skills
```

Comment:
I guess the ASF URL shortener is reliable but I would still rather use JQL to be really sure links don't break later. I don't think the URL shortener makes much difference inside a markdown link.

Reply:
Done